decidim-comments/config/locales/en.yml

```diff
@@ -15,9 +15,7 @@ en:
           user_group_id:
             label: Comment as
         opinion:
-          against: I am against
-          in_favor: I am in favor
-          neutral: I am neutral
+          neutral: Neutral
```

Comment:
When adding translations to comments (that go inside the components) we need to rebuild the bundle. Also, add the catalan & spanish translations since they also need to be at the bundle.